test/factories/episode_image_factory.rb

```diff
@@ -11,10 +11,6 @@
     format { "png" }
     status { "complete" }
 
-    after(:create) do |episode_image, _evaluator|
-      episode_image.url = episode_image.published_url if episode_image.status == "complete"
```

Comment:
This is handled by the overridden image_file.url method now.  (Same as media_resources).